oioswift/proxy/controllers/account.py

```diff
@@ -56,6 +56,25 @@ def get_response_headers(info):
     return resp_headers
 
 
+def account_listing_bucket_response(account, req, response_content_type,
+                                    listing=None):
+    if response_content_type != 'application/json':
+        # AWS S3 is always call wit format=json
+        # check method GET in ServiceController (swift3/controllers/service.py)
+        return HTTPPreconditionFailed(body='Invalid content type')
+
+    data = []
+    for entry in listing:
+        data.append({'name': entry['name'], 'count': entry['objects'],
+                     'bytes': entry['bytes'],
+                     'last_modified': Timestamp(entry['mtime']).isoformat})
+    account_list = json.dumps(data)
+    ret = HTTPOk(body=account_list, request=req, headers={})
```

Comment:
Shouldn't we encode `account_list` before passing it as the response body? Some parts of the code encode bodies with `ascii` charset, but here `utf-8` seems more appropriate (because we explicitly set the response charset).

Reply:
The `json` from `swift.common.utils` is the standard `python json` module and it use the default encoding `utf-8`. I don't see value to enforce `utf-8` after the `dumps`.
Let's just make clear that encoding `utf-8` is used.